### DISCUSSION TITLE
Accepting typing-extensions 4.x

### DISCUSSION THREAD

**Discussion by pawamoy** [collerek/ormar#728] [General]
Hello, any reason you exclude typing-extensions 4.x from the accepted range?

https://github.com/collerek/ormar/blob/3ac3b5066e37592316c1d9aabb0b43acb6f0f71e/pyproject.toml#L63-L64

It could help dependencies resolution if Ormar accepted versions 4.x, though maybe Ormar indeed *doesn't* work with such versions.

**Comment by collerek** [comment]
I think it was caused by poetry deps conflict. 
Will check.

**Comment by collerek** [comment]
Or not :D (or something else already updated in the meantime).
I bumped the version - it will be up in the next release.

Do you need this urgently?

**Reply by pawamoy** [reply]
No no, don't worry, it's already fantastic that you updated this so quickly, thanks a lot 🙂
